status/projects.py

```diff
@@ -177,8 +177,7 @@ def project_summary_data(self, row):
                     row.value['contact'] = ord_det['fields']['project_lab_name'] + ': ' + ord_det['fields']['project_lab_email']
                 elif row.value['contact']  == ord_det['fields']['project_pi_email']:
                     row.value['contact'] = ord_det['fields']['project_pi_name'] + ': ' + ord_det['fields']['project_pi_email']
-
-        if row.key[0] in ['review', 'ongoing', 'reception control'] and 'queued' in row.value:
+        if row.key[0] in ['review', 'ongoing', 'reception control', 'open'] and 'queued' in row.value:
```

Comment:
```suggestion
        # The status "open" is added here since this method is reused with only the statuses open/closed. 
        if row.key[0] in ['review', 'ongoing', 'reception control', 'open'] and 'queued' in row.value:
```